### PR TITLE
RELATED: FET-723 avoid lodash/get in the rest of the packages

### PR DESCRIPTION
**Pull request by no23reason**
Remove `lodash/get` from `api-client-bear` and most of `sdk-ui-ext`.

Also add ESLint rule to improve the feedback in IDEs (they show ESLint errors, not dependabot ones).

Also fix one really old bug (see the second commit, this dates all the way back to app-components). 

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by no23reason** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3474/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5107/ : SUCCESS in 12m 20s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/883/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3475/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/883/ : SUCCESS in 15m 34s

File: libs/api-client-bear/src/execution/experimental-executions.ts [hunk 1]
```diff
@@ -70,30 +68,30 @@ const getBaseMetricTitle = partial(getMetricTitle, "");
 
 const CONTRIBUTION_METRIC_FORMAT = "#,##0.00%";
 
-function getPoPDefinition(measure: IMeasure) {
-    return get(measure, ["definition", "popMeasureDefinition"], {});
+function getPoPDefinition(measure: any) {
+    return measure?.definition?.popMeasureDefinition ?? {};
```
Comment (kandl) [inline libs/api-client-bear/src/execution/experimental-executions.ts:72]: I assume we don't have better typings/type guards for these methods?

**Review by kandl — COMMENTED** [review]
Is it safe to replace lodash get with nullish coalescing? (lodash get does not default null values)

**Review by kandl — APPROVED** [review]
LGTM

File: libs/api-client-bear/src/execution/experimental-executions.ts (hunk 1 above)
Comment (no23reason) [reply libs/api-client-bear/src/execution/experimental-executions.ts:72]: We don't unfortunately :-/

**Comment by no23reason** [comment]
> Is it safe to replace lodash get with nullish coalescing? (lodash get does not default null values)

Good point, hopefully no one depended on this behavior as it is itself a bit strange (differentiating between null and undefined which is something we should avoid if at all ossible in the codebase).

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1316

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1033/ : FAILURE in 2m 17s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1303/ : SUCCESS in 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/16/ : SUCCESS in 10m 05s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5116/ : SUCCESS in 12m 00s

**Pull request merged**